db/version_edit.cc

```diff
@@ -172,6 +174,16 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
         char p = static_cast<char>(1);
         PutLengthPrefixedSlice(dst, Slice(&p, 1));
       }
+      if (f.priv_meta != nullptr) {
+        PutVarint32(dst, CustomTag::kPrivMeta);
+        // EncodePrivateMetadata's implementation takes care of tracking the
+        // size of the metadata it is storing since this size varies. We store 1
+        // referring to the number of chunks to respect the decoding
+        // implementation
+        char p = static_cast<char>(1);
+        PutLengthPrefixedSlice(dst, Slice(&p, 1));
+        f.EncodePrivateMetadata(dst);
```

Comment:
Sure. I'll return the string from  f.EncodePrivateMetadata and then append it to dst. I'll resubmit tomorrow.
Is there any other thing that I should change? Being in different timezones makes communication a bit asynchronous :)
Thanks